### PR TITLE
[ebay.be] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/issues/13603

### CODE REVIEW

File: src/chrome/content/rules/ebay.be.xml [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+	<target host="www.sellvrfn.befr.ebay.be" />
+	<target host="settings.befr.ebay.be" />
+	<target host="shiptrack.befr.ebay.be" />
+	<target host="signin.befr.ebay.be" />
+	<target host="sofe.befr.ebay.be" />
+	<target host="spd.befr.ebay.be" />
+	<target host="www.sps.befr.ebay.be" />
+	<target host="srtm.befr.ebay.be" />
+	<target host="viv.befr.ebay.be" />
+	<target host="wantitnow.befr.ebay.be" />
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.be.xml]: `benl.ebay.be`

File: src/chrome/content/rules/ebay.be.xml [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="eBay.be">
+
+	<target host="advertising.ebay.be" />
+	<target host="auth.ebay.be" />
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.be.xml]: `befr.ebay.be`

**Comment by youdly** [comment]
Added `befr` and `benl`.

**Comment by J0WI** [comment]
Looks like we need to redirect them to `www`, just like `^`.
Travis does also report an incomplete cert chain issue, but I was not able to reproduce this.

**Comment by youdly** [comment]
Added redirect

**Pull request merged**